src/index/index.module.css

```diff
@@ -179,9 +182,9 @@
 }
 .cases__case {
   display: grid;
-  gap: 2rem 2rem;
+  gap: 2rem;
   padding: 2rem 0;
-  grid-template-columns: minmax(20rem, 2fr) minmax(23rem, 1fr);
+  grid-template-columns: minmax(15rem, 2fr) minmax(20rem, 1fr);
```

Comment:
Jeg forstår ikke helt hvorfor disse verdiene er valgt 🤔 Er det meningen at venstre kolonne skal være mindre enn høyre, eller er det noe jeg misforstår? 
Kunne det like gjerne vært `grid-template-columns: 1fr 1fr;`?

Reply:
Godt spørsmål. Regner med at intensjonen er å la bildet (som er på venstre side i dette tilfellet) shrinke mer enn teksten, trolig for å holde teksten mer leslig. Men kan godt hende vi likegodt kan bruke lik bredde. Kan teste og se hva jeg liker best.

Reply:
Lar det stå som det var. Blir penest når bildene vokser mer enn teksten (2fr vs 1fr) på store skjermer, men er nødvendig at teksten har en viss minstestørrelse for å være lesbar i praksis.